rasa/core/training/data.py

```diff
@@ -15,7 +15,9 @@ def __init__(
     def limit_training_data_to(self, max_samples: int) -> None:
         self.X = self.X[:max_samples]
         self.y = self.y[:max_samples]
-        self.true_length = self.true_length[:max_samples]
+        self.true_length = (
+            self.true_length[:max_samples] if self.true_length is not None else None
```

Comment:
or an empty list?

Reply:
it's set to `None` by default, didn't want to mess with it so kept it that way